### PR TITLE
docs: less green on localhost

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview
Changed things around a little after realizing green was showing up on code playgrounds

change local host color settings

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.4.1-alpha.8b4b7d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6140bec387d2d75e9ed205fb--gamut-preview.netlify.app>https://6140bec387d2d75e9ed205fb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6140bec387d2d75e9ed205fb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.5.1-alpha.e84775.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6140c08226780c67947ffd66--gamut-preview.netlify.app>https://6140c08226780c67947ffd66--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6140c08226780c67947ffd66>Deploy Logs</a></p>

File: packages/styleguide/.storybook/theme.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { create } from '@storybook/theming';
 import { theme as gamutTheme, colors } from '@codecademy/gamut-styles/src';
 import logo from './assets/logo.svg';
 
-const isLocalhost = globalThis.location?.toString().includes('localhost');
+const isLocalHost = globalThis.location?.toString().includes('localhost');
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/.storybook/theme.ts:5]: lol, I've always thought of `localhost` as one word - like `Localhost`...

**Review by JoshuaKGoldberg — APPROVED** [review]
💚

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.5.1-alpha.6c15e3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6140c790d861a2009b8d6f74--gamut-preview.netlify.app>https://6140c790d861a2009b8d6f74--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6140c790d861a2009b8d6f74>Deploy Logs</a></p>

File: packages/styleguide/.storybook/theme.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ export const theme = create({
   colorSecondary: colors.navy,
 
   // UI
-  appBg: isLocalhost ? colors['green-0'] : colors.white,
-  appContentBg: isLocalhost ? colors['green-0'] : colors.white,
+  appBg: isLocalhost ? colors['paleBlue'] : colors.white,
```
Comment (codecaaron) [inline packages/styleguide/.storybook/theme.ts:18]: I don't think this does anything.

**Review by codecaaron — APPROVED** [review]

File: packages/styleguide/.storybook/theme.ts (hunk 2 above)
Comment (dreamwasp) [reply packages/styleguide/.storybook/theme.ts:18]: oh weird, it is working for me (making the sidebar blue)  but it does sometimes require a reset/`yarn build-all`. 
should i remove it?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.5.1-alpha.146ce0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6140e7fc4fdfe21a53287be1--gamut-preview.netlify.app>https://6140e7fc4fdfe21a53287be1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6140e7fc4fdfe21a53287be1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@44.5.3-alpha.9bade1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6141f2879997504199356984--gamut-preview.netlify.app>https://6141f2879997504199356984--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6141f2879997504199356984>Deploy Logs</a></p>

**Pull request merged**